### PR TITLE
Alkwa/update UI library version

### PR DESCRIPTION
**Pull request by alkwa-msft**
# What
Merging in changes we would normally handle in automation into 1.0.1-beta.2 branch.

### CODE REVIEW

**Review by prprabhu-ms — APPROVED** [review]

File: packages/communication-react/CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ This log was last generated on Mon, 24 Jan 2022 23:18:53 GMT and should not be m
 
 <!-- Start content -->
 
-## [1.0.1](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.0.1)
+## [1.0.1-beta.2](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.0.1-beta.2)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:7]: I think this should be `-beta.1`

Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.md:7]: Similarly the compare link bwelow should be 1.0.0...1.0.1-beta.1

Comment (prprabhu-ms) [reply packages/communication-react/CHANGELOG.md:7]: Err... why? We're releasing `1.0.1-beta.2` next, eh?

Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.md:7]: Oh this is into the release branch not main! my bad ignore me ^_^

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**